streetcrm/admin.py

```diff
@@ -723,7 +723,8 @@ class ParticipantAdmin(mixins.AdminArchiveMixin, mixins.SignInSheetAdminMixin, S
                      "participant_zipcode_address")
     list_filter = (admin_filters.ArchivedFilter,)
     list_display = ("name", "US_primary_phone", "institution", "participant_street_address", "participant_city_address",)
-    readonly_fields = ("action_history", "event_history_name")
+    # HAD TO REMOVE event_history_name DUE TO ISSUES
+    readonly_fields = ("action_history",)
```

Comment:
In general, we try to use some searchable string across changes like this -- maybe `TODO`?